sotodlib/coords/planets.py

```diff
@@ -297,6 +305,58 @@ def get_source_pos(source_name, timestamp, site='_default'):
     ra, dec, distance = amet0.radec()
     return ra.to(units.rad).value, dec.to(units.rad).value, distance.to(units.au).value
 
+
+def get_source_azel(source_name, timestamp, site='_default'):
+    """
+    Get the apparent azimuth and elevation of a celestial source at a specific timestamp and observing site.
+
+    This function calculates the apparent azimuth and elevation of a celestial source, accounting for the geographical position
+    on Earth but assuming no atmospheric refraction. Note that it will download a 16M ephemeris file on first use.
+
+    Args:
+        source_name: Planet name; in capitalized format, e.g. "Jupiter"
+        timestamp (float): The Unix timestamp representing the time for 
+        which to calculate azimuth and elevation.
+        site (str or so3g.proj.EarthlySite): if this is a string, the
+        site will be looked up in so3g.proj.SITES dict.
+
+    Returns:
+      az (float): in radians.
+      el (float): in radians.
+      distance (float): in AU.
+    """
+    # Get the ephemeris -- this will trigger a 16M download on first use.
+    de_url = RESOURCE_URLS['de421.bsp']
+    de_filename = au_data.download_file(de_url, cache=True)
+
+    planets = jpllib.SpiceKernel(de_filename)
+    for k in [
+            source_name,
+            source_name + ' barycenter',
+    ]:
+        try:
+            target = planets[k]
+            break
+        except (ValueError, KeyError):
+            pass
+    else:
+        options = list(planets.names().values())
+        raise ValueError(f'Failed to find a match for "{source_name}" in '
+                         f'ephemeris: {options}')
+
+    if isinstance(site, str):
+        site = so3g.proj.SITES[site]
+
+    observatory = site.skyfield_site(planets)
+
+    timescale = skyfield_api.load.timescale()
+    sf_timestamp = timescale.from_datetime(
+        datetime.datetime.fromtimestamp(timestamp, tz=skyfield_api.utc))
+    amet0 = observatory.at(sf_timestamp).observe(target)
+    el, az, distance = amet0.apparent().altaz()
+    return az.to(units.rad).value, el.to(units.rad).value, distance.to(units.au).value
```

Comment:
Up to this point, this function is basically identical to get_source_pos.  Please either:
- move the duplicated code into a helper function, for both get_source_pos and get_source_azel to use, OR
- simply add an argument to get_source_pos that returns az,el instead of ra,dec (and you can have get_source_azel be an alias that calls into get_source_pos)